index.html

```diff
@@ -2,16 +2,23 @@
 layout: base
 ---
 <h1>Solar Forecast Arbiter</h1>
+<h2><em>A paradigm shift in forecast evaluation</em></h2>
 
+<br>
 <div class="row">
 <img class="homepage-image myb-3" src="/images/homepage_presentation.png">
 </div>
-<p>The Solar Forecast Arbiter is an open source tool to enable evaluations of solar irradiance, solar power, and net-load forecasts that are impartial, repeatable and auditable. The Solar Forecast Arbiter supports the <a href="http://www.energy.gov/solar-office">U.S. Department of Energy Solar Energy Technologies Office</a> Solar Forecasting 2 program and the broader solar forecast community. </p>
+
+<br>
+
+<p>The Solar Forecast Arbiter supports a paradigm shift in the way we evaluate solar and net-load forecasts. For too long we have been unclear with each other and even with ourselves about what, exactly, we are evaluating. Whether you're forecast user, vendor, or researcher, the Solar Forecast Arbiter will force you to think about your analysis and what really matters before you start parsing data and calculating statistics. This can be an unfamiliar and uncomfortable process. Stick with it and we think you'll see the value in this approach. <b>So what's the value? Clear, transparent communication of the skill of a precisely defined forecast.</b></p>
```

Comment:
```suggestion
<p>The Solar Forecast Arbiter supports a paradigm shift in the way we evaluate solar and net-load forecasts. For too long we have been unclear with each other and even with ourselves about what, exactly, we are evaluating and how the evaluation is done. Whether you're a forecast user, vendor, or researcher, the Solar Forecast Arbiter will force you to think about your analysis and what really matters before you start parsing data and calculating statistics. This can be an unfamiliar and uncomfortable process. Stick with it and we think you'll see the value in this approach. <b>So what's the value? Clear, transparent communication of the skill of a precisely-defined forecast.</b></p>
```